kzg_prover/bin/gen_verifier.rs

```diff
@@ -73,6 +72,10 @@ fn main() {
     );
     assert!(result.is_ok());
 
+    let result_unwrapped = result.unwrap();
+    result_unwrapped.0.expect("prover should not fail");
+    let _advice_polys = result_unwrapped.1.clone();
```

Comment:
_advice_polys is not used

Reply:
It removed at https://github.com/summa-dev/summa-solvency/commit/6db5fb7cc56192741c86d55ff8f59c9eb7dea05b